extensions/amp-story/1.0/amp-story.css

```diff
@@ -880,11 +880,13 @@ amp-story .amp-video-eq,
   /* 224px = the width of a narrow phone (320px) - tap-safe margins (48px * 2) */
   min-width: 224px !important;
   max-width: 400px !important;
-  /* Properties overrideable by user */
-  --interactive-accent-color: #005AF0;
-  --interactive-background-color: white;
+	/* Properties overrideable by user */
+	--interactive-accent-color: #005AF0;
+	--interactive-background-color: white;
+	--interactive-prompt-background-color: var(--interactive-accent-color);
+	--interactive-prompt-text-color: white;
+	--interactive-prompt-alignment: left;
```

Comment:
Added some user-overrideable properties according to Hong's directions for the prompt mostly